measures/TimeseriesCSVExport/tests/timeseries_csv_export_test.rb

```diff
@@ -107,7 +107,7 @@ def test_tmy_runperiod
     args_hash['reporting_frequency'] = 'RunPeriod'
     args_hash['include_enduse_subcategories'] = 'false'
     args_hash['output_variables'] = 'Surface Outside Normal Azimuth Angle, Surface Window Heat Gain Rate'
-    expected_values = { 'EnduseTimeseriesLength' => 1, 'EnduseTimeseriesWidth' => @@include_enduse_subcategories[args_hash['include_enduse_subcategories']] + 71 }
+    expected_values = { 'EnduseTimeseriesLength' => 1, 'EnduseTimeseriesWidth' => @@include_enduse_subcategories[args_hash['include_enduse_subcategories']] + 93 }
```

Comment:
What columns does this add to enduse_timeseries.csv?

Reply:
All of the window shading surfaces get additional outputs because of requesting the output variables Surface Outside Normal Azimuth Angle' and 'Surface Window Heat Gain Rate'. I assume there is no way to exclude them...?